crates/dreamchecker/src/lib.rs

```diff
@@ -562,7 +562,6 @@ pub struct AnalyzeObjectTree<'o> {
     impure_procs: ViolatingProcs<'o>,
     waitfor_procs: HashSet<ProcRef<'o>>,
 
-    sleeping_overrides: ViolatingOverrides<'o>,
```

Comment:
The way this was implemented wasn't effective and the cause of the misses because it didn't catch overrides that called that called procs that slept.